contract_compassion/migrations/1.3/pre-migration.py

```diff
@@ -0,0 +1,30 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Albert SHENOUDA <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+import logging
+logger = logging.getLogger()
+
+
+def update_analytic_accounts(cr):
+    """ Change the type of the analytic accounts. """
+    # Find all analytic accounts related to events
+    cr.execute(
+        "DELETE *"
+        "FROM ir.model.data"

```

Comment:
Missing a space after ir.model.data otherwise it will do "...FROM ir.model.dataWHERE model"